### ISSUE TITLE
Please update

### ISSUE THREAD

**Issue by MaynardMiner** [GPUOpen-LibrariesAndSDKs/display-library#3]
ADL2_OverdriveN_Capabilities_Get does not work. That's just one of the buggy things with this repository.

I have seen references to:

ADL2_OverdriveN_CapabilitiesX2_Get, etc. but this method is not in this library.

**Comment by AshotN** [comment]
The docs seem to be riddled with bugs and poor examples, and they are outdated. Sad excuse for documentation by AMD

**Comment by MaynardMiner** [comment]
Yep...I had to run their software through a debugger to figure out the little I needed to know regarding gathering fan speeds through their ADL.

It is fairly disappointing.
